src/theme/typography.ts

```diff
@@ -77,10 +77,10 @@ const margin: Margin = {
 
 const createLinks = (themeColors: typeof colors): Links => ({
   primary: {
-    color: '#0275d8',
-    hoverColor: '#015399',
-    activeColor: '#0275d8',
-    focusBgColor: '#ebf4fc',
+    color: themeColors.brand[600],
+    hoverColor: themeColors.brand[800],
+    activeColor: themeColors.brand[600],
+    focusBgColor: themeColors.brand[50],
```

Comment:
Please use `themeColors.primary` instead of the `brand` I fixed the confusion in the ticket description.